core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -152,102 +144,18 @@ public void init() throws StoreInitializationException {
     }
   }
 
-  public static void addSecurityConfigsToClientProperties(SchemaRegistryConfig config,
-                                                          Properties props) {
-    addSslConfigsToClientProperties(config, props);
-    addSaslConfigsToClientProperties(config, props);
+  public static void addSchemaRegistryConfigsToClientProperties(SchemaRegistryConfig config,
+                                                                Properties props) {
+    props.putAll(config.originalProperties());
```

Comment:
Is this actually right? Aren't the non-`kafkastore.` prefixed configs for https access to SR itself whereas the prefixed ones are for connecting to Kafka?